crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -15,7 +15,7 @@ use stdarch_test::assert_instr;
 #[inline]
 #[target_feature(enable = "neon,sha3")]
 #[cfg_attr(test, assert_instr(eor3))]
-#[unstable(feature = "stdarch_neon_sha3", issue = "117225")]
+#[stable(feature = "stdarch_neon_sha3", since = "1.78.0")]
```

Comment:
@Amanieu do I need to bump this to 1.79.0 now?

Also seems like I need to merge/rebase?

Reply:
You can do this when the FCP ends in 2 weeks.

Reply:
@tarcieri You should update the version to 1.79 now that FCP is done.

Reply:
Done